src/App.js

```diff
@@ -23,29 +111,46 @@ export function App() {
     }
   }
   
+  function showPage() {
+    setShown(true);
+  }
+  
+  function hidePage() {
+    setShown(false);
+  }
+  
   return (
-    <div className="App">
-      <nav>
-        <ul>
-          <li><p>RAID</p></li>
-          <li><a href='default.asp'>Search!</a></li>
-          <input type="text" id="searchValue" placeholder="Search" class="search" onChange={searchChangeHandler}/>
-        </ul>
-      </nav>
-      <Banner/>
-      { beingSearched === true ? 
-      (<RowRetry title="Search Results" 
-      fetchURL={"/search/multi?api_key="+API_KEY+"&language=en-US&query="+searchTerm+"&page=1&include_adult=false"}
-      isLargeRow/>)
-      : null}
-      <RowRetry title="NETFLIX ORIGINALS" fetchURL={requests.fetchNetlfixOriginals} isLargeRow/>
-      <RowRetry title="Trending Now" fetchURL={requests.fetchTrending}/>
-      <RowRetry title="Top Rated" fetchURL={requests.fetchTopRated}/>
-      <RowRetry title="Action Movies" fetchURL={requests.fetchActionMovies}/>
-      <RowRetry title="Comedy Movies" fetchURL={requests.fetchComedyMovies}/>
-      <RowRetry title="Horror Movies" fetchURL={requests.fetchHorrorMovies}/>
-      <RowRetry title="Documentaries" fetchURL={requests.fetchDocumentaries}/>
-    </div>
+    <div>
+    {appShown === true ? (
+        <div className="App">
+          <nav>
+            <ul>
+              <li><p>RAID</p></li>
+              <li><a href='default.asp'>Search!</a></li>
+              <input type="text" id="searchValue" placeholder="Search" class="search" onChange={searchChangeHandler}/>
+            </ul>
+          </nav>
```

Comment:
Needs to be added `<Logout hidePage={hidePage}/>` below `</nav>` can we add after the merge issue resolved?